### ISSUE TITLE
NTR: Ureteric stricture

### ISSUE THREAD

**Issue by peupeubangbang** [obophenotype/human-phenotype-ontology#10958]
**Preferred term label:**
Ureteric stricture

**Synonyms**
Ureteral stricture

**Definition (free text, please give PubMed ID)**
Obstruction of urine flow through one or both ureters caused by issues such as injury, infection, tumours and surgery. In rare instances there can be congenital strictures.

**Comment**
This is separate to Ureteral stenosis (HP:0000071) which should be used for congenital narrowing of the ureter(s).

**Parent term (use hpo.jax.org/app)**
Ureteral obstruction HP:0006000

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@peupeubangbang 
Adding new term: 
- ID: HP:6001215
- label: Ureteric stricture
- parents: [Abnormal ureter morphology (HP:0025633)](https://hpo.jax.org/app/browse/term/HP:0025633)
- pmid: PMID:25100183
- def: Obstruction of urine flow through one or both ureters caused by issues such as injury, infection, tumours and surgery.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10958
- synonyms:
